pom.xml

```diff
@@ -185,6 +185,11 @@
         <artifactId>jackson-annotations</artifactId>
         <version>${jackson-annotations.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Why do you need to import `jackson` jar?

Reply:
I just found a bug that, if we don't import jackson jar, we would get the error that jackson class is not found, when using submarine-all-*.jar to submit a submarine job.